docs/content/deployment/guides/docker.mdx

```diff
@@ -81,9 +81,13 @@ Dagit servers expose a health check endpoint at `/dagit_info`, which returns a J
 }
 ```
 
+---
+
 ## Multi-container Docker deployment
 
-More advanced dagster deployments will require deploying more than one container. For example, if you are using [dagster-daemon](/deployment/dagster-daemon) to run schedules and sensors or manage a queue of runs, you'll likely want a separate container running the `dagster-daemon` service. This service must have access to your `dagster.yaml` and `workspace.yaml` files, just like the Dagit container. You can also configure your [workspace](/concepts/code-locations/workspace-files) so that your code can be updated and deployed separately in its own container running a gRPC server, without needing to redeploy the other dagster services. To enable this setup, include a container exposing a gRPC server at a port, and add that port in your `workspace.yaml` file.
+More advanced Dagster deployments will require deploying more than one container. For example, if you are using the [Dagster daemon](/deployment/dagster-daemon) to run schedules and sensors or manage a queue of runs, you'll likely want a separate container running the `dagster-daemon` service.
+
+This service must have access to your `dagster.yaml` and `workspace.yaml` files, just like the Dagit container. You can also configure your [code location server](/concepts/code-locations/workspace-files) so that your code can be updated and deployed separately in its own container running a gRPC server, without needing to redeploy the other Dagster services. To enable this setup, include a container exposing a gRPC server at a port and add that port in your `workspace.yaml` file.
```

Comment:
In this passage you have a "code location server" "running a gRPC server" which isn't really right. 

I think in this context it is right to say the code location IS a grpc server.